zed_wrapper/src/nodelet/src/zed_wrapper_nodelet.cpp

```diff
@@ -140,19 +140,20 @@ namespace zed_wrapper {
                 mZedParams.svo_input_filename = mSvoFilepath.c_str();
                 mZedParams.svo_real_time_mode = false;
             } else  if (!mRemoteStreamAddr.empty()) {
-                if (mVerMajor > 2 || (mVerMajor == 2 && mVerMinor >= 8)) {
-                    std::vector<std::string> configStream = sl_tools::split_string(mRemoteStreamAddr, ':');
-                    sl::String ip = sl::String(configStream.at(0).c_str());
+#if ((ZED_SDK_MAJOR_VERSION>2) || (ZED_SDK_MAJOR_VERSION==2 && ZED_SDK_MINOR_VERSION>=8) )
+                std::vector<std::string> configStream = sl_tools::split_string(mRemoteStreamAddr, ':');
+                sl::String ip = sl::String(configStream.at(0).c_str());
 
-                    if (configStream.size() == 2) {
-                        mZedParams.input.setFromStream(ip, atoi(configStream.at(1).c_str()));
-                    } else {
-                        mZedParams.input.setFromStream(ip);
-                    }
+                if (configStream.size() == 2) {
+                    mZedParams.input.setFromStream(ip, atoi(configStream.at(1).c_str()));
                 } else {
-                    ROS_ERROR_STREAM("To acquire a remote stream is required at least the ZED SDK v2.8");
-                    return;
+                    mZedParams.input.setFromStream(ip);
                 }
+
+#else
+                ROS_ERROR_STREAM("To acquire a remote stream is required at least the ZED SDK v2.8");
```

Comment:
Maybe something like instead? "Acquiring a remote stream requires the ZED SDK v2.8 or newer"

Reply:
Done